e2e/pages/events/createOrderEvent.page.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
       id: '#interimEndDate_interimEndDate',
       options: {
         endOfProceedings: 'At the end of the proceedings',
-        namedDate: 'On a named date',
+        namedDate: 'At the end of a named date',
```

Comment:
Might be worthwhile adding the other option here even if it isn't being used in the test yet.

Reply:
yup good idea will add :)